src/EnergyPlus/UnitarySystem.hh

```diff
@@ -424,6 +431,24 @@ namespace UnitarySystems {
         Real64 m_minWaterLoopTempForHR; // water coil heat recovery loops
         bool m_waterSideEconomizerFlag; // user input to enable lockout with economizer
         bool m_WaterHRPlantLoopModel;   // signifies water heat recovery loop for this CoilSystem
+        std::array<int, 4> m_OAMixerNodes{0, 0, 0, 0};
+        Real64 m_CoolOutAirVolFlow;
+        Real64 m_CoolOutAirMassFlow;
+        Real64 m_HeatOutAirVolFlow;
+        Real64 m_HeatOutAirMassFlow;
+        Real64 m_NoCoolHeatOutAirVolFlow;
+        Real64 m_NoCoolHeatOutAirMassFlow;
+        Real64 m_HeatConvTol;
+        Real64 m_CoolConvTol;
+        int m_HVACSizingIndex;
+        int m_AvailStatus;
+        bool m_IsZoneEquipment;
+        bool m_ZoneCompFlag;
+        std::string m_AvailManagerListName;
+        int m_ZoneUnitaryNum;  // 1-based index of this parent type for zone equipment processing
+        int m_PTACNum;         // 1-based index of this parent type for zone equipment processing
+        int m_PTHPNum;         // 1-based index of this parent type for zone equipment processing
+        int m_PTWSHPNum;       // 1-based index of this parent type for zone equipment processing
```

Comment:
These 4 (m_ZoneUnitaryNum through m_PTWSHPNum) are for SysAvailManagers. This was my first attempt at storing the relative index to each Eq type. e.g., 12 UnitarySystems of different types but only 3 PTACs, with PTAC index 1, 2 and 3. This concept should be able to be stored in a single variable for all Eq types, ZoneEqCompNum or something like that.